src/device.c

```diff
@@ -467,15 +474,12 @@ int device_clear_credentials_on_file(void)
 int device_start_event(void)
 {
 	int rc;
-
-	rc = event_start(on_event_timeout);
+	rc = event_start(on_event_timeout, thing.time_sec);
 	if (rc < 0) {
```

Comment:
what does rc means? And this zero, what does the zero means? I think the number zero should be replaced by a variable with a explicit name